### PR TITLE
Refact/keyp 312 314 channel control

### PR DESCRIPTION
**Pull request by jeanpierrefortune**

### CODE REVIEW

File: java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java [hunk 1]
```diff
@@ -116,18 +115,12 @@ public final void clearObservers() {
...
-                // close the physical channel thanks to CLOSE_AFTER flag
-                processSeRequest(null, CLOSE_AFTER);
-                logger.trace(
-                        "Explicit communication closing requested, starting removal sequence.");
-            } catch (KeypleReaderException e) {
-                logger.error("KeypleReaderException while terminating. {}", e.getMessage());
-            }
-        } else {
-            logger.trace("Explicit physical channel closing already requested.");
+        if (logger.isTraceEnabled()) {
```
Comment (jeanpierrefortune) [inline java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java:118]: Check this please

File: java/component/keyple-core/src/test/java/org/eclipse/keyple/core/seproxy/plugin/AbsObservableLocalReaderTest.java [hunk 2]
```diff
@@ -205,25 +206,35 @@ public void states() throws Exception {
...
+        // initiate the SE removal sequence
         r.notifySeProcessed();
-        verify(r, times(1)).processSeRequest(null, ChannelControl.CLOSE_AFTER);
+
+        // assert currentState have changed
+        Assert.assertEquals(AbstractObservableState.MonitoringState.WAIT_FOR_SE_REMOVAL,
+                r.getCurrentMonitoringState());
     }
 
+    @Ignore
```
Comment (jeanpierrefortune) [inline java/component/keyple-core/src/test/java/org/eclipse/keyple/core/seproxy/plugin/AbsObservableLocalReaderTest.java:225]: Check this

File: java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java (hunk 1 above)
Comment (pterr) [reply java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java:118]: Olivier, I let you check this method

Comment (odelcroi) [reply java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java:118]: From what I understand, the method should remains the same

**Comment by jeanpierrefortune** [comment]
Replaced by #373

**Pull request closed without merging**